compiler/README.md

```diff
@@ -0,0 +1,56 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the trained model. It takes many model formats and converts to a common 'Circle' format which is then deployed. This compiler runs on the target device and targets supported are ExynosXXX.
```

Comment:
> This compiler runs on the target device and targets supported are ExynosXXX.

I think you wanted to say that `circle` model is run on target device, not compiler itself.

Reply:
Right now, can the compiled 'circle' (cross compiled on my desktop for arm) alone be moved inside XU4 and executed for inference? if so, any guide on how to do it? [I thought the compiler and back end is inseparable]. if what you say is right, then ONERT alone can be run on XU4 fed with circle.

Reply:
@mhs4670go , can you please add some short explanation about runninng compiler output in _onert_ ?

> cross compiled on my desktop for arm

We usually run compiler in host x86-64,
- `tf2tfliteV2.py` will convert TF model to TFlite model
- `tflite2circle` will convert TFlite model to circle model
- `circle2circle` will do some optimization (not much as of today)
- (@mhs4670go , please edit this line for `nnpackager`)

And optionally with quantization
- `record-minmax` will add quantization information into circle model
- `circle-quantizer` will quantize to UINT8 model

Reply:
@btheosam You can see that guide [here](https://github.sec.samsung.net/STAR/nnfw/blob/master/docs/nnfw/howto/HowToRunNnpackage.md). 
And we have a test(`tf2circle-value-pbtxt-remote-test`) which tests from tensorflow model to runtime. Though this module's been deprecated, you can get some information with its `README.md`.

Reply:
@mhs4670go Got it , thanks.